frontend/src/components/VideoQualityLabel/VideoQualityLabel.tsx

```diff
@@ -0,0 +1,16 @@
+import type { VideoQualityLabelProps } from './VideoQualityLabel.types';
+import styles from './VideoQualityLabel.module.css';
```

Comment:
its not something really important, but JFI, there is a good practice to sort imports, usually its configired in ESLint rules and does it automatically onSave, but since we don't have it, it would be nice at least to separate them by type: vendors, components, utils and styles